### ISSUE TITLE
no more quests

### ISSUE THREAD

**Issue by tomboy-64** [endless-sky/endless-sky#766]
I've been playing for a few days now. In the beginning I got a 3 recon missions and met the Drak somewhere in the south-east. Since then I've not come across any other missions all over the map. I wonder if I ended up in some kind of deadlock?
Playing 0.8.11 on Linux.

**Comment by natangold** [comment]
This question is more suited to the [forum](https://groups.google.com/forum/#!forum/endless-sky) than here. This is meant as a place for Michael Z to keep track of bugs, glitches, and feature requests. That said, (a) the only major plot line fully implemented is based in Southern Space, and (b) you pick up the vast majority of plot missions by visiting the spaceport (with the 'P' key) on various planets. For best results, cruise the Southern Rim and visit the spaceport on each planet. (Oh, also the plot doesn't really kick in until a certain hard-coded date: you'll know it when you land on a planet and the conversation panel immediately comes up with "The moment you step off your ship you realize something is very wrong…")

**Comment by jafdy** [comment]
Although the wording in the comment shows that people are getting confused between the Qurg and the Drak. It may be that the Drak are just an invention of the Qurg. The Qurg are still the Qurg.

**Comment by masterpierround** [comment]
The Drak have not been added to the game yet. They are coming soon. You have met the Quarg. Also, what do you mean by "recon missions"? Could you explain what you had to do for these?

**Comment by tomboy-64** [comment]
I was sent around by some Militia Lady to take not of pirate activity in a couple of systems, then meet up in another system. I had 3 of those missions in the first game I played. In the new game, I haven't had any missions after the initial tutorial with the retiree.
And I indeed had met the Quarg. I confused them with the Drak, which he briefly mentioned.

**Comment by endless-sky** [comment]
The rest of the story line missions don't become available until the civil war begins (July 4th 3014).

There are a few other minor mission threads, plus a bunch of one-off missions that you'll receive if you visit enough spaceports.

**Comment by tomboy-64** [comment]
It's Aug 27th 3016, and there is no civil war.

**Comment by tomboy-64** [comment]
Okay, to clarify this. I was able to perform the missions Pact Recon 0 - 3 from "free worlds intro.txt" once.
I'm also able to trigger "First Contact: Quarg" from "quarg missions.txt" in all games so far.
Same with the "intro missions.txt" - I'm able to play all missions from there.

I do also hit the one occurence @natangold mentions, where "The moment you step off your ship you realize something is very wrong...". Apart from that, not a single mission or story line element.

I've built it myself on gentoo. This is the [build-script](https://github.com/tomboy-64/gentoo/blob/endless-sky/games-strategy/endless-sky/endless-sky-0.8.11.ebuild), if someone cares to take a look.
This means I would also be able to build it with debugging options enabled, if someone could give me a hint what to look for.
